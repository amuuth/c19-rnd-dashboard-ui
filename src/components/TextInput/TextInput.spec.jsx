import React from 'react'
import { shallow } from 'enzyme'
import { TextInput } from '.'

it('should render input', () => {
  const wrapper = shallow(<TextInput />)
  expect(wrapper).toMatchSnapshot()
})

it('should render the input with especified props', () => {
  const props = {
    placeholder: 'Search something ...',
  }
  const wrapper = shallow(<TextInput {...props} />)
  expect(wrapper.find('input').props().placeholder).toEqual(props.placeholder)
})
