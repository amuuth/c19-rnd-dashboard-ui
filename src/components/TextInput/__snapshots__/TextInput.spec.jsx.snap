// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render input 1`] = `ShallowWrapper {}`;
